src/main/java/com/jfrog/ide/common/nodes/ApplicableIssueNode.java

```diff
@@ -1,28 +1,12 @@
 package com.jfrog.ide.common.nodes;
```

Comment:
This comment doesn't belong here but to LicenseViolationNode. I noticed in the [jfrog-idea-plugin PR](https://github.com/jfrog/jfrog-idea-plugin/pull/360/files#diff-e8c86d269eb7869e86bf48358ca362df71873ac49beeeeb17e5c70b4be456d5aR214) that all of the words in the titles of the new issues are capitalized, and I think it's better than the title of LicenseViolationNode, where only the first word is capitalized. Could you change it there to be like yours?